### DISCUSSION TITLE
v.1.0.0

### DISCUSSION THREAD

**Discussion by hbeni** [hbeni/fgcom-mumble#154] [General]
With the release of [Mumble 1.4.0](https://www.mumble.info/blog/mumble-1.4.230/) I'm proud to also release the first production version of FGCom-Mumble. :partying_face: :tada: 

After the first drafts started in Jun 2020, the project underwent alot of development and followed the "release early, release often" approach to faciliate early testing. That helped not only to iron out bugs, but also yielded early support in ATC-Pie, a widely used ATC client in the flightsim world. the project also contributed in the development and testing of mumbles plugin framework, and the lua-mumble library along the way.
Now, 654 commits later, FGCom-mumble hopefully can contribute to the flightsim realism in providing a simulator agnostic radio simulation.

Thank you everyone who contributed and tested along the way!
I hope we will see more releases with new features, so keep ideas floating into the issue tracker :)

<hr /><em>This discussion was created from the release <a href='https://github.com/hbeni/fgcom-mumble/releases/tag/v.1.0.0'>v.1.0.0</a>.</em>

**Comment by mill-j** [comment]
Very good. Congratulations!

**Comment by D-MKF1** [comment]
Hi, can't install it on macOS Monterey Version 12.1
Error message in mumble 1.4.230: not possible to install fgcom-mumble-macOS.bundle
Any idea?
Cheers,
Marc

**Reply by hbeni** [reply]
Hm, thats bad.
I think the only solution is to self compile - I have only very limited means to do testing on mac (ie only the build process here on github).

If you can provide improvements, that would be awesome!

**Reply by hbeni** [reply]
@D-MKF1 I rechecked. The release is built with macOS 11.6.
My impression was that this should be compatible - but did the clib change incompatible in 12?
What happens when you try a manual install of the plugin file (unzip the bundle file and move the compiled plugin lib file into mumbles plugin directory)?
But probably the easiest way is to just fetch the source and build the plugin yourself; that will ensure compatibility. You can refer to the github actions workflow definition, and also the makefile client/mumble-plugin/Makefile (target `plugin-macOS`) on how I let build it (i have no mac machine here and am stuck to github actions to test).

```
2022-03-07T01:13:12.1176880Z ##[group]Operating System
2022-03-07T01:13:12.1177930Z macOS
2022-03-07T01:13:12.1178140Z 11.6.4
2022-03-07T01:13:12.1178330Z 20G417
2022-03-07T01:13:12.1178550Z ##[endgroup]
2022-03-07T01:13:12.1178800Z ##[group]Virtual Environment
2022-03-07T01:13:12.1179090Z Environment: macos-11
2022-03-07T01:13:12.1179330Z Version: 20220220.1
2022-03-07T01:13:12.1179770Z Included Software: https://github.com/actions/virtual-environments/blob/macOS-11/20220220.1/images/macos/macos-11-Readme.md
2022-03-07T01:13:12.1180350Z Image Release: https://github.com/actions/virtual-environments/releases/tag/macOS-11%2F20220220.1
```

**Comment by D-MKF1** [comment]
Perhaps it will be a solution. I’m on 12.2.1 macOS Monterey.
I just can't get around to trying it out at the moment. I'll try it over the weekend.
Thanks very much,
Marc

**Reply by hbeni** [reply]
Did you get results yet, marc?
